owasp-top10-2016-mobile/m2/cool_games/mobile/lib/views/register_screen.dart

```diff
@@ -10,7 +11,8 @@ class RegisterScreen extends StatelessWidget {
 
   _register(BuildContext context, String username, String password) async {
     // set up POST request arguments
-    String url = 'http://10.0.2.2:9051/register';
+    String host = Platform.isAndroid ? "10.0.2.2" : "localhost";
+    String url = 'http://$host:9051/login';
```

Comment:
I believe this might be a bug, would you mind changing the route to `register` here?
What do you think, @gustavocovas?

Reply:
Oops, sorry. Of course. Changed.